build/teamcity/cockroach/nightlies/sqllogic_corpus_nightly_impl.sh

```diff
@@ -75,7 +75,7 @@ if [ $exit_status = 0 ]; then
 fi
 
 # Generate a corpus for all mixed version variants
-for config in local-mixed-22.1-22.2; do
+for config in local-mixed-22.2-23.1; do
```

Comment:
Ugh, this is an existing issue 🤦 , but there is a missing done on this loop. The corpus is generated but it messes up the build process status

Reply:
fixed